packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -677,7 +684,25 @@ export class MarketOperationUtils {
                 wholeOrderPrice,
             );
         }
-        return { ...optimizerResult, quoteReport };
+
+        let priceComparisonsReport: PriceComparisonsReport | undefined;
+        if (_opts.shouldIncludePriceComparisonsReport) {
```

Comment:
Can this be bumped to a static function like the `_computeQuoteReport`

Reply:
I think a simple Assertible test is good enough, I set one up with some jq magic to count number of items in `priceComparisons` where price is **not** null (`.priceComparisons.jq('[.[] |select(.price != null)] | length')`). We can then just assert that there's at least 1 valid price in the response 👍